Formula/squashfs.rb

```diff
@@ -24,8 +25,10 @@ class Squashfs < Formula
   end
 
   def install
+    args = "XATTR_SUPPORT=0 EXTRA_CFLAGS=-std=gnu89 LZO_SUPPORT=1 LZO_DIR='#{HOMEBREW_PREFIX}' XZ_SUPPORT=1 XZ_DIR='#{HOMEBREW_PREFIX}"
```

Comment:
For long argument lists, we prefer arrays instead of strings, like the following that @dunn suggested. It helps with readability, and if the arguments are changed in the future, the diffs and commit history are more useful, since you have one line per argument.

```
args = %W[
  XATTR_SUPPORT=0
  EXTRA_CFLAGS=-std=gnu89
  LZO_SUPPORT=1
  LZO_DIR=#{HOMEBREW_PREFIX}
  etc...
]
args << "LZ4_SUPPORT=1" if build.with? "lz4"
```